agent-jenkins-tools/Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN git clone -b allow_self_signed_certs https://github.com/mikenairn/python-jen
     pushd python-jenkins && \
     yum -y --enablerepo=extras install epel-release && \
     yum -y install python-pip && \
+    yum -y install skopeo && \
```

Comment:
It's not a big deal, but you can merge this line and the previous line to make `yum -y install python-pip skopeo` if you want.

Reply:
Makes sense, I'll update now, test and merge 👍